skimage/transform/histogram_matching.py

```diff
@@ -48,12 +53,10 @@ def match_histograms(image, reference, multichannel=False):
 
     References
     ----------
-    .. [1] http://paulbourke.net/miscellaneous/equalisation/
+     .. [1] http://paulbourke.net/miscellaneous/equalisation/
```

Comment:
```suggestion
    .. [1] http://paulbourke.net/miscellaneous/equalisation/
```
I'm sorry, I don't know why my previous suggestion added that space :S

Reply:
I accepted your earlier suggestion and did not do any change in this line, so it seems you are correcting your earlier suggestion... :]